src/Message/CtaUrl/ImageHeader.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Netflie\WhatsAppCloudApi\Message\CtaUrl;
+
+class ImageHeader extends Header
```

Comment:
I made the decision a few versions ago to declare all classes as final.

This achieves safer, faster and easier updates since we prevent anyone from extending classes that are implementation details and that library clients should not extend.